include/fw_info.h

```diff
@@ -98,8 +132,10 @@ OFFSET_CHECK(struct fw_info, firmware_address,
  * ID/version combination.
```

Comment:
Comment on a description and  name of this type:
It seems we spend most of the documentation on fw_info_abi talking about what this type does *not* include in long  related sentences. I would name this a "header" in the type name or use some other naming to emphasize that this is not the complete structure for the ABI. 

Alternatively the documentation could state something like this
This struct serves as the header for an ABI, containing only the basic information about the ABI without describing the  implementation details dependent on an ID and version combination.

Reply:
This has been rewritten, take another look if you haven't already.